packages/manager/apps/public-cloud/src/index.js

```diff
@@ -31,10 +31,11 @@ import 'ovh-ui-kit/dist/oui.css';
 import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.css';
 
 import atInternet from './components/at-internet';
+import easterEggs from './components/easter-eggs';
 import preload from './components/manager-preload';
 
-import 'bootstrap-tour/build/css/bootstrap-tour.min.css';
-import './assets/theme/index.less';
+import './assets/theme/default/index.less';
+import './assets/theme/dark/index.less';
```

Comment:
Can we import that dynamically if user choose to enable the dark mode ?

Reply:
Didn't find yet a proper way to achieve this. Any help would be greatly appreciated.

Reply:
@antleblanc i think one way to achieve this would be to use ocLazyLoad for your component and import the less files inside your component. That way the css will be injected when your component will be dynamically loaded.
I did not test it but i think it should work that way

Reply:
@frenautvh addressed with ec3409e.